index.js

```diff
@@ -88,11 +88,12 @@ module.exports = {
     },
     {
       files: ['*.ts'],
-      extends:
-      [
+      extends: [
```

Comment:
•100% ➜ eslint . --fix-dry-run
Error: ESLint configuration in sparkpost/index is invalid:
        - Unexpected top-level property "overrides[1].extends".

Referenced from: /Users/gwalls/workspace/eslint-config-sparkpost/api.js
Referenced from: /Users/gwalls/workspace/metrics-api/.eslintrc.json
    at validateConfigSchema (/Users/gwalls/workspace/metrics-api/node_modules/eslint/lib/config/config-validator.js:235:15)
    at Object.validate (/Users/gwalls/workspace/metrics-api/node_modules/eslint/lib/config/config-validator.js:261:5)
    at loadFromDisk (/Users/gwalls/workspace/metrics-api/node_modules/eslint/lib/config/config-file.js:544:19)
    at load (/Users/gwalls/workspace/metrics-api/node_modules/eslint/lib/config/config-file.js:587:20)
    at /Users/gwalls/workspace/metrics-api/node_modules/eslint/lib/config/config-file.js:453:36
    at Array.reduceRight (<anonymous>)
    at applyExtends (/Users/gwalls/workspace/metrics-api/node_modules/eslint/lib/config/config-file.js:431:26)
    at loadFromDisk (/Users/gwalls/workspace/metrics-api/node_modules/eslint/lib/config/config-file.js:551:22)
    at load (/Users/gwalls/workspace/metrics-api/node_modules/eslint/lib/config/config-file.js:587:20)
    at /Users/gwalls/workspace/metrics-api/node_modules/eslint/lib/config/config-file.js:453:36